sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -515,3 +515,82 @@ ALTER TABLE `api_token` ADD `token_api` varchar(40);
 #IfMissingColumn api_token patient_id
 ALTER TABLE `api_token` ADD `patient_id` bigint(20) NOT NULL;
 #EndIf
+
+#IfNotTable fhir_forced_id
+CREATE TABLE `fhir_forced_id` (
```

Comment:
Once this table is up and running this would be a great use case for `redis`, if it's still in the architecture.
Of course things will work fine with a two-step lookup as well.

Reply:
With doctrine, should be but, i'm not sure. If not, we can implement.